charts/calico/templates/calico-node-rbac.yaml

```diff
@@ -176,6 +178,47 @@ rules:
 
 ---
 
+#CNI cluster role
```

Comment:
nit, but could you keep the comment styling consistent with the others?

basically:

```
# CNI cluster role
```

instead of 

```
#CNI cluster role    
```